index.md

```diff
@@ -0,0 +1,30 @@
+# Signalen
+
+Signalen makes it easier for residents to make a nuisance report, and helps the city respond to the most important things first.
+
+Signalen includes:
+
+* algorithmic processing by type of complaint and second layer triaging and prioritisation by humans
+* automatic updates to complainants about the status of their report
+* automatic alerts to city staff based on analysis of combined reports
+* easy passing on of reports to other city workers and subcontractors
+
+## The Signalen team
+
+To make Signalen fully reusable outside of Amsterdam, [VNG Realisatie](https://www.vngrealisatie.nl/) is working together with Den Bosch, Almere and Amsterdam, and supported by the [Foundation for Public Code](https://publiccode.net/). The team will start work in spring 2020.
```

Comment:
As Amsterdam has done all the development to this date, can we raise the prominence of Amsterdam in this sentence? Something which highlights that it's being brought from Amsterdam to all the others?